assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -189,15 +208,4 @@
 			padding: 0 10px;
 		}
 	}
-
-	.googlesitekit-header__children {
```

Comment:
Re-ordered this selector inside `.googlesitekit-header` to satisfy the [no-descending-specificity](https://stylelint.io/user-guide/rules/list/no-descending-specificity/) rule.